.travis/run_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+
+API_URL=https://api.github.com/repos/PBSPro/pbspro
+
+PR_NUM=${1:-""}
+
+cd test/fw
+pip install -r requirements.txt .
+pbs_config --make-ug
+cd ../tests
+
+if [ -z ${PR_NUM} ]; then
+	changed_files=$(git status -s . | awk '{print $2}' | grep -v "pbs_smoketest.py")
+else
+	changed_files=$(curl --silent --insecure ${API_URL}/pulls/$1/files | grep '"filename": "test/tests/' | grep -v "pbs_smoketest.py" | cut -d \" -f4)
+fi
+
+if [ ! -z ${changed_files} ]; then
+	for f in ${changed_files}
+	do
+		if [ ! -f ${f} ]; then
+			continue
+		fi
+		ts_name=$(grep -oP 'class[\s]+(\K\w+)(?=\(.*\):)' ${f})
```

Comment:
@hirenvadalia : Adding 'Test' ( As per our coding guidelines test suite name should start with "Test" prefix) keyword to regex will more accurate results. Otherwise if the test script has any other class it will try to run it.